crates/astria-conductor/src/executor.rs

```diff
@@ -117,21 +113,26 @@ struct Executor<C> {
     /// so that we can mark the block as final on the execution layer when
     /// we receive a finalized sequencer block.
     sequencer_hash_to_execution_hash: HashMap<Hash, Vec<u8>>,
+
+    /// Chose to execute empty blocks or not
+    disable_empty_block_execution: bool,
 }
 
 impl<C: ExecutionClient> Executor<C> {
-    async fn new(mut execution_rpc_client: C, chain_id: ChainId) -> Result<(Self, Sender)> {
+    async fn new(mut execution_rpc_client: C, conf: &Config) -> Result<(Self, Sender)> {
```

Comment:
IMO it's nicer to not pass the conf around as a "grab back". I prefer the former version (we are not doing a good job of adhering to this in other parts of the code base either).

Reply:
Put this back to how it was before.